src/Main/Main.make

```diff
@@ -250,6 +250,9 @@ prepare: $(APPNAME)
 	chmod +x $(BASE_DIR)/Setup/Linux/usr/bin/$(APPNAME)-uninstall.sh
 	cp $(BASE_DIR)/License.txt $(BASE_DIR)/Setup/Linux/usr/share/doc/$(APPNAME)/License.txt
 	cp $(BASE_DIR)/../doc/html/* "$(BASE_DIR)/Setup/Linux/usr/share/doc/$(APPNAME)/HTML"
+	mkdir -p $(BASE_DIR)/Setup/Linux/usr/share/veracrypt/languages
+	cp -r $(BASE_DIR)/../Translations/* $(BASE_DIR)/Setup/Linux/usr/share/veracrypt/languages/
+	cp $(BASE_DIR)/Common/Language.xml $(BASE_DIR)/Setup/Linux/usr/share/veracrypt/languages/Language.en.xml
```

Comment:
There no need to copy Language.en.xml since it is already embedded in the code and it will be used as fallback automatically.

Reply:
Both done